core/layouts/geary/layout--geary.tpl.php

```diff
@@ -31,82 +31,92 @@
 
   <?php if ($content['header']): ?>
     <header class="l-header" role="banner" aria-label="<?php print t('Site header'); ?>">
-      <div class="container container-fluid">
-        <div class="row">
-          <?php print $content['header']; ?>
-        </div><!-- /.row -->
+      <div class="l-header-inner container container-fluid">
+        <?php print $content['header']; ?>
```

Comment:
Figured out that full width rows don't need to have `.row` and `.col-xs-12` wrappers. Just having container will work will.